examples/webxr_xr_dragging_custom_depth.html

```diff
@@ -429,19 +429,19 @@
 
 						isDepthSupplied = true;
 
-					});
+					} );
 
-				} else if (! renderer.xr.hasDepthSensing() && isDepthSupplied) {
+				} else if ( ! renderer.xr.hasDepthSensing() && isDepthSupplied ) {
 
-					group.children.forEach(child => {
+					group.children.forEach( child => {
 
 						child.material.uniforms.depthWidth.value = 0;
 						child.material.uniforms.depthHeight.value = 0;
 						child.material.uniforms.NeedUpdate = true;
```

Comment:
Would you mind removing these lines, see https://github.com/mrdoob/three.js/pull/27695#discussion_r1624051890.

Reply:
> Would you mind removing these lines, see [#27695 (comment)](https://github.com/mrdoob/three.js/pull/27695#discussion_r1624051890)?

Done.